pkg/api/v1beta1/dynakube/validation/oneagent.go

```diff
@@ -17,14 +18,18 @@ const (
 `
 	errorImageFieldSetWithoutCSIFlag = `The DynaKube's specification tries to enable ApplicationMonitoring mode and get the respective image, but the CSI driver is not enabled.`
 
-	errorNodeSelectorConflict = `The DynaKube's specification tries to specify a nodeSelector conflicts with an another Dynakube's nodeSelector, which is not supported.
-The conflicting Dynakube: %s
-`
+	errorNodeSelectorConflict = `The DynaKube specification attempts to deploy a %s, which conflicts with another DynaKube's %s. Only one per namespace is supported.
```

Comment:
```suggestion
	errorNodeSelectorConflict = `The DynaKube specification attempts to deploy a %s, which conflicts with another DynaKube's %s. Only one Agent per node is supported.
```

Reply:
Minor nitpick to add Agent here

The other thing is that only one agent is allowed per node and not namespace, or am I wrong there?

Reply:
Nice catch, done